packages/ckeditor5-package-generator/lib/templates/ts-legacy/package.json

```diff
@@ -46,13 +42,14 @@
     "@ckeditor/ckeditor5-dev-build-tools": "<%= packageVersions.ckeditor5DevBuildTools %>",
     "@ckeditor/ckeditor5-inspector": ">=<%= packageVersions.ckeditor5Inspector %>",
     "@ckeditor/ckeditor5-package-tools": "<%= packageVersions.packageTools %>",
-    "@typescript-eslint/eslint-plugin": "~5.43.0",
-    "@typescript-eslint/parser": "^5.18.0",
+    "@typescript-eslint/parser": "^8.32.1",
     "@vitest/coverage-istanbul": "^2.0.5",
     "@vitest/browser": "^2.0.5",
     "ckeditor5": "latest",
-    "eslint": "^7.32.0",
+    "eslint": "^9.27.0",
     "eslint-config-ckeditor5": ">=<%= packageVersions.eslintConfigCkeditor5 %>",
+    "eslint-plugin-ckeditor5-rules": ">=<%= packageVersions.eslintConfigCkeditor5 %>",
```

Comment:
The version for `eslint-plugin-ckeditor5-rules` package we should read from the `eslint-plugin-ckeditor5-rules` package.

Do we really want to use `>=` range instead of `^`?

Reply:
`>=` was here to force using the latest version when a package was created, for example, a year ago.

Reply:
That's a valid point. ESLint releases were mostly compatible with the previous versions until version 9.

@przemyslaw-zan, could we use `^` instead of `>=` for linter packages?